website/static/js/comment.js

```diff
@@ -62,6 +115,58 @@ var exclusifyGroup = function() {
     }
 };
 
+ko.subscribable.fn.convertHtmlToMarkdown = function(commentContent) {
+    return ko.computed(function() {
```

Comment:
I guess it's not a generic HTML to Markdown converter, we're just looking at the links and `<br>`s. Maybe there's a better name.